test/model/PolyhedronMetricTest.cpp

```diff
@@ -0,0 +1,74 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+#include <array>
+#include "polyhedralGravity/model/Polyhedron.h"
+#include "polyhedralGravity/model/PolyhedronDefinitions.h"
+#include "polyhedralGravity/model/GravityModel.h"
+#include "polyhedralGravity/util/UtilityConstants.h"
+
+class PolyhedronMetricTest : public ::testing::Test {
+protected:
+    const std::vector<polyhedralGravity::Array3> _cubeVertices{
+                    {-1.0, -1.0, -1.0},
+                    {1.0, -1.0, -1.0},
+                    {1.0, 1.0, -1.0},
+                    {-1.0, 1.0, -1.0},
+                    {-1.0, -1.0, 1.0},
+                    {1.0, -1.0, 1.0},
+                    {1.0, 1.0, 1.0},
+                    {-1.0, 1.0, 1.0}
+    };
```

Comment:
I don't quite recall if we had also some a bit less regular tests? Just asking because often scaling errors may disappear for exactly the special case of a lot of values being 1, so usually would also have a test with some multiple. (or just moving the points to [0,2] even)

Reply:
Good point! For this case, it absolutely makes sense to shift the points a bit, as a "standard" cube is not required and to make the test case less trivial 👍

Regarding the gravity model, we have tests going way beyond the trivial cases.